apps/next-evans/lib/db/friends.ts

```diff
@@ -49,7 +49,7 @@ async function getFriendsFromDB(lang: Lang): Promise<Record<string, Friend>> {
           invariant(firstEdition !== undefined);
           invariant(firstEdition.edition_impressions !== null);
           const newestEdition = doc.editions.find(
-            (e) => e.type === mostModernEditionType(doc.editions.map((ed) => ed.type)),
+            (edition) => edition.type === mostModernEditionType(doc.editions),
```

Comment:
see comment in `lib/editions.ts` below. this simplification was made possible by that type change.